apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -170,48 +257,149 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
-                    <div className="step-connector">
-                        <div className="step-line"></div>
-                    </div>
-                </div>
-                <div className="stepper">
-                    <div className="step-number">2</div>
-                    <div className="step-text">
-                        <h5>{ t(translateKey + "modals.verify.heading") }</h5>
+                        <h5 className=" text-center" > { t(translateKey + "modals.verify.heading") }</h5>
+
                         <Segment basic className="pl-0">
                         {
                             error
                                 ? (
-                                <>
-                                    <Message
-                                        error data-testid={ `${ testId }-code-verification-form-field-error` }>
+                                    <Message 
+                                        className="totp-error-message" 
+                                        data-testid={ `${ testId }-code-verification-form-field-error` }>
                                         { t(translateKey + "modals.verify.error") }
                                     </Message>
-                                </>
                                 )
                                 : null
                         }
-                        <Forms
-                            data-testid={ `${ testId }-code-verification-form` }
-                            onSubmit={ (values: Map<string, string>) => {
-                                verifyCode(values.get("code"));
-                            } }
-                            submitState={ submit }
-                        >
-                            <Field
-                                data-testid={ `${ testId }-code-verification-form-field` }
-                                name="code"
-                                label={ t(translateKey + "modals.verify.label") }
-                                placeholder={ t(translateKey + "modals.verify.placeholder") }
-                                type="text"
-                                required={ true }
-                                requiredErrorMessage={ t(translateKey + "modals.verify.requiredError") }
-                            />
-                        </Forms>
+                            <Form onSubmit={ (event: React.FormEvent<HTMLFormElement>): void => {
+                                            handleTOTPVerificationCodeSubmit(event);
+                            } }>
+                                <Container>
+                                    <Grid className="totp">
+                                        <Grid.Row textAlign="center" centered columns={ 6 } >
+                                            <Grid.Column >
+                                                <Form.Field >
+                                                    <input autoFocus ref= { pinCode1 } name="pincode-1" 
```

Comment:
Check other places as well.

```suggestion
                                                    <input
                                                        autoFocus
                                                        ref= { pinCode1 }
                                                        name="pincode-1" 
```